README.md

```diff
@@ -27,9 +28,8 @@ You can however set it to `rmw_fastrtps_dynamic_cpp` using the environment varia
 
 ## Advance usage
 
-[//]: # (TODO sloretz - link ROS 2 discovery documentation when it's created)
 ROS 2 only allows for the configuration of certain middleware features.
-For example, see [ROS 2 QoS policies](https://docs.ros.org/en/foxy/Concepts/About-Quality-of-Service-Settings.html#qos-policies).
+For example, see [ROS 2 QoS policies](https://docs.ros.org/en/rolling/Concepts/About-Quality-of-Service-Settings.html#qos-policies).
```

Comment:
foxy is E.O.L, we do not want to set the link for that.

(future enhancement note) branch name can be replaced with github pre-commit hook so that we can have an appropriate URL depends on branch name.